### DISCUSSION TITLE
Unable to dump NES carts

### DISCUSSION THREAD

**Discussion by ZFuriax** [sanni/cartreader#458] [Need help]
Just got my Cart Reader (V3-Alter).  I flashed it to firmware 9.4.  I've got The Adventure of Link connected to the cart reader through an NES-to-SNES adapter.  The main menu of the cart reader shows:

OPENSOURCE CARTREADER
- Add-ons
- Super Nintendo
- Mega Drive
- Nintendo 64
- Game Boy
- About
- Reset

I believe I'm supposed to go to Add-Ons -> Consoles -> NES/Famicom.  When I do it says Searching Database for half a second and then shows a menu of # A -> Z ? 

No matter what I select the device says:

Please Wait.. 
Database file not found

Press Button...

What am I doing wrong?  I can't seem to find any assistance in the wiki or elsewhere.

**Comment by sanni** [comment]
Did you copy the database file to the SD card?

![image](https://dl.dropboxusercontent.com/s/v8il10n64mo6r2w/sdcopy.jpg?dl=1)

**Comment by ZFuriax** [comment]
Nope.  I must've missed that step.  Looks like it's working now.  Thanks!

**Comment by SAVE-THE-HERO** [comment]
Thank you sanni-san

**Comment by PW5190** [comment]
Does the OSCR support dumping unlicensed NES cartridges (i.e. Tengen, Camerica/Codemasters (silver and gold) (and Aladdin Deck Enhancer (with detachable cartridges)), Color Dreams/Wisdom Tree (various colors (most notably black and blue)), Little Red Hood (requires a licensed game to lock on), Action 52 (translucent plastic))?

**Reply by nsx0r** [reply]
Aladdin dumping confirmed working
![image](https://user-images.githubusercontent.com/13054953/226952152-73ff9518-9f33-4216-a3ec-eef43d6d6a8b.png)

**Comment by sanni** [comment]
Here is the list of the currently [supported mappers](https://github.com/sanni/cartreader/blob/master/Cart_Reader/NES.ino#L28).
The mapper is the first number in each line. 

To know what mapper a cartridge uses you can check this [database](http://nes.dnsabr.com/)

For example [Action 52 uses Mapper 228](http://nes.dnsabr.com/entity/view?id=30). Mapper 228 is not in the list above so it's not supported.

**Comment by nsx0r** [comment]
I added a script for mapper 228 (Action 52, Cheetahmen 2) but I cannot test it because I don't own these games (yet).
